Mathlib/Tactic/Qify.lean

```diff
@@ -61,7 +61,7 @@ macro_rules
 | `(tactic| qify $[[$simpArgs,*]]? $[at $location]?) =>
   let args := simpArgs.map (·.getElems) |>.getD #[]
   `(tactic|
-    simp (config := {decide := false}) only [zify_simps, qify_simps, push_cast, $args,*]
+    simp -decide only [zify_simps, qify_simps, push_cast, $args,*]
```

Comment:
I guess we could just remove this now? But maybe this should happen elsewhen.

Reply:
Yeah, the `-decide` isn't necessary here anymore. I just didn't want to change the logic with this PR.